pkgs/by-name/co/comfyui/package.nix

```diff
@@ -0,0 +1,164 @@
+{
+  lib,
+  python312,
+  linkFarm,
+  writers,
+  writeTextFile,
+  fetchFromGitHub,
+  stdenv,
+  symlinkJoin,
+  config,
+  modelsPath ? "/var/lib/comfyui/models",
+  inputPath ? "/var/lib/comfyui/input",
+  outputPath ? "/var/lib/comfyui/output",
```

Comment:
Since we don't really test other combinations of those settings, I am not sure how will this will be tested. Maybe at should just use a to directory prefix?

Reply:
So just `/var/lib/comfyui/` would be configurable and the directories would be hardcoded relative to that? That would be more rigid than comfyui allows us but I can change it if that's a Nix limitation

Reply:
Yep, like that. It is not really a limitation but it adds lots of variability and testing all that can be hard. Whatever you prefer.